uw-frame-components/portal/settings/partials/user-settings.html

```diff
@@ -15,6 +15,9 @@
         <div ng-if='NOTIFICATION.enabled'>
           <button class="btn btn-flat" ng-click="resetKey(KV_KEYS.DISMISSED_NOTIFICATION_IDS,'DISMISSED_NOTIFICATION_IDS')">Reset dismissed notifications <i class='fa fa-spin fa-circle-o' ng-if='DISMISSED_NOTIFICATION_IDS'></i></button>
         </div>
+        <div ng-if='MISC_URLS.resetLayoutURL'>
```

Comment:
I ❤️  the graceful degradation (no `resetLayoutURL`? No worries.) and that the URL config *is* the config for enabling or disabling the feature (rather than having to coordinate a parallel `.enabled` flag). I feel like we're iteratively getting better and better at this kind of grace, and it's wonderful.